common/coins_test.go

```diff
@@ -29,7 +29,7 @@ func TestEtherAmount(t *testing.T) {
 
 func TestToDecimals(t *testing.T) {
 	val := NewEtherAmount(123456)
-	require.Equal(t, fmt.Sprint(val.ToDecimals(5)), "1.23456")
+	require.Equal(t, "1.23456", FmtFloat(val.ToDecimals(5)))
 	val = NewEtherAmount(1234567890)
-	require.Equal(t, fmt.Sprint(val.ToDecimals(6)), "1234.56789")
```

Comment:
The expected/actual values here were reversed and it seemed like as good a place as any to use the `FmtFloat` function added by this PR.  I've never used `fmt.Sprint` like above, but I imagine it behaves like `"%v"` in a formatted string, which in some cases prints undesirable exponent notation.